src/mrpro/__init__.py

```diff
@@ -1,2 +1,10 @@
+from pathlib import Path
```

Comment:
suggestion:
create a version.py file with the content and import its __version__ in the init
as it currently is (sorry, was my suggestion), Path etc will end up in mrpro namespace

Reply:
Do you want to rename `VERSION` to `version.py` or `_version.py` and define the version only there? 

Or do you want to add an additional `_version.py` file, where we put the code that is currently in the main `__init__.py`, so that we can import `__version__` from there?

Both should work, but for the first option we need to adapt the workflow as well.

Reply:
The latter.
I would leave VERSION as a text file, as is.
I would create an additional _version.py and move the version parsing logic there and import it in init.
such that the only change in this pr to init is:
`from _version import __version__`